components/metadata-service/internal/externalapi/servicedetailsvalidation_test.go

```diff
@@ -289,12 +324,35 @@ func TestServiceDetailsValidator_API_OAuth(t *testing.T) {
 		// when
 		err := validator.Validate(serviceDetails)
 
+		// then
+		assert.NoError(t, err)
+	})
+
+	t.Run("should not accept Basic Auth credentials with empty basic", func(t *testing.T) {
```

Comment:
We should not allow to register service with both authorization methods provided (I was able to register api with both oauth and basic auth)